test/functional/p2p_blocksonly.py

```diff
@@ -51,13 +40,13 @@ def run_test(self):
 
         self.log.info('Check that txs from rpc are not rejected and relayed to other peers')
         assert_equal(self.nodes[0].getpeerinfo()[0]['relaytxes'], True)
-        txid = self.nodes[0].testmempoolaccept([sigtx])[0]['txid']
+
+        assert_equal(self.nodes[0].testmempoolaccept([tx_hex])[0]['allowed'], True)
```

Comment:
99791e7560d40ad094eaa73e0be3987581338e2d: If you touch this, I'd prefer to remove it. This doesn't check anything